### PR TITLE
US 1889327: Updated H2 and column headers.

### PR DESCRIPTION
**Pull request by poojapoojari**
Created 8th PR for user story-1889327. Updated H2 headings and column headers.
User story link: [https://dev.azure.com/mseng/TechnicalContent/_queries/edit/1889327/](url)

### CODE REVIEW

**Comment by poojapoojari** [comment]
Hi @adegeo, 

I've updated the H2 headings and column headers for the next 20 files and created this pull request for your review. If you have questions or feedback, please add it as a comment in the PR. The standard review period is five business days. 

Thank you, 
Pooja

File: dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-seek-a-storyboard-synchronously.md [hunk 1]
```diff
@@ -12,12 +14,12 @@ ms.assetid: 03e06271-a946-4810-88ea-3fb4cfa9e0f1
...
 The following example shows how to use the <xref:System.Windows.Media.Animation.Storyboard.SeekAlignedToLastTick%2A> method of a <xref:System.Windows.Media.Animation.Storyboard> to seek to any position in a storyboard animation synchronously.  
   
-## Example  
+## XAML markup
  The following is the XAML markup for the sample.  
   
  [!code-xaml[SeekStoryboard_snip#SeekStoryboardSynchronouslyExampleWholePage](~/samples/snippets/csharp/VS_Snippets_Wpf/SeekStoryboard_snip/CSharp/SeekStoryboardSynchronouslyExample.xaml#seekstoryboardsynchronouslyexamplewholepage)]  
   
-## Example  
+## XAML code
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-seek-a-storyboard-synchronously.md:22]:
```suggestion
## Code
```

File: dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-seek-a-storyboard.md [hunk 2]
```diff
@@ -12,12 +14,12 @@ ms.assetid: 887bb39a-0c2a-4ae8-956d-1d9f6f8ebbfc
...
 The following example shows how to use the <xref:System.Windows.Media.Animation.Storyboard.Seek%2A> method of a <xref:System.Windows.Media.Animation.Storyboard> to jump to any position in a storyboard animation.  
   
-## Example  
+## Sample XAML markup
  Below is the XAML markup for the sample.  
   
  [!code-xaml[SeekStoryboard_snip#SeekStoryboardExampleWholePage](~/samples/snippets/csharp/VS_Snippets_Wpf/SeekStoryboard_snip/CSharp/SeekStoryboardExample.xaml#seekstoryboardexamplewholepage)]  
   
-## Example  
+## Code with XAML
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-seek-a-storyboard.md:22]:
```suggestion
## Code
```

**Review by adegeo — APPROVED** [review]
Thanks!

**Pull request merged**